### PR TITLE
suppermatter

### PR DESCRIPTION
**Pull request by SonixApache**
Finally, a supermatter-themed food!
Comes in normal and exciting red variants. They're animated and glow.
suppermatter
Normal one is simple to make; caramel cake + 5 plasma + 5 nitrogen.
Exciting one requires is: caramel cake + 5 plasma + 5 oxygen.
That one's harder because plasma and oxygen mix into dexalin if you try to add it from a single beaker.

 - [ ] update wiki

🆑 
 - rscadd: You can now make suppermatter in your microwaves. One caramel cake, 5 plasma, and 5 nitrogen or oxygen. Engineers will love it, but your teeth will hate it!

### CODE REVIEW

**Comment by brndd** [comment]
The normal one is clearly lemon flavoured and the exciting one is peach flavoured.

Not a big fan of the recipes because they just use a bunch of chemicals instead of botany-related stuff like the kitchen should be using.

**Comment by SonixApache** [comment]
Lemon could work, don't think we have peaches though.

**Comment by DrSnips** [comment]
We have several dishes that aren't reliant on botany. Why does this one need to be?

**Comment by RealestEstate** [comment]
would the plasma and oxygen not mix when inside of the microwave to create dexalin though?
Also I vote to rename it to superbatter

**Comment by Kurfursten** [comment]
I believe realest is right. Test.

**Comment by SonixApache** [comment]
Hm, I did test it in an upgraded microwave. I'll test in a normal one as well.

**Comment by Kurfursten** [comment]
No need to test, I checked the code. Apparently microwaves have been NOREACT for 9 years.

Who knew? I wouldn't have even guessed NOREACT was that old.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -5947,3 +5947,55 @@ obj/item/weapon/reagent_containers/food/snacks/butterfingers_l
...
 	bitesize = 2
+
+/obj/item/weapon/reagent_containers/food/snacks/sliceable/suppermatter
+	name = "suppermatter"
+	desc = "Extremely dense and powerful food."
+	slice_path = /obj/item/weapon/reagent_containers/food/snacks/suppermattershard
+	storage_slots = 1
+	slices_num = 4
+	icon_state = "suppermatter"
+	w_class = W_CLASS_MEDIUM
```
Comment (Kurfursten) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5958]: This needs a special eatverb "slams"

I'd like it to slam into their face but eatverb can't support complex phrasing

Comment (SonixApache) [reply code/modules/reagents/reagent_containers/food/snacks.dm:5958]: `"Grey Mcshirt slams the suppermatter."`
Sounds kinda weird to me.

**Pull request merged**